packages/material-ui/src/ExpansionPanelSummary/ExpansionPanelSummary.js

```diff
@@ -17,7 +17,7 @@ export const styles = (theme) => {
       display: 'flex',
       minHeight: 8 * 6,
       transition: theme.transitions.create(['min-height', 'background-color'], transition),
-      padding: theme.spacing(0, 3),
+      padding: theme.spacing(0, 2),
```

Comment:
Why this change? It's a pretty old component so I'd be careful with breaking changes here.

Reply:
The whole point is described here #20580

Reply:
Sure, why not.